### ISSUE TITLE
Focus' custom tab opens in a separate task

### ISSUE THREAD

**Issue by CoderSpinoza** [mozilla-mobile/focus-android#2203]
Currently, custom tabs with Focus for Android opens in a separate task, unlike other browsers such as Chrome, Samsung Internet Browser, and Firefox for Android. It is almost same as just opening a full browser from user's perspective.

I am not sure if current implementation is by design, but It might hinder user experience as stated in [Chrome Custom Tabs documentation](https://developer.chrome.com/multidevice/android/customtabs), as well as provides inconsistency with other browser's custom tabs functionality

> Both options present challenges — launching the browser is a heavy context switch that isn't customizable, while WebViews don't share state with the browser and add maintenance overhead.

Therefore, it would be much better if Focus for Android also opens a custom tab in the same task as the application.

**Comment by pocmo** [comment]
Fixing this depends on #770

**Comment by lobontiumira** [comment]
I was not able to reproduce this issue on the latest Focus Nightly, Focus Beta, nor RC builds.
Closing the ticket.
